loraxDataTest/CladesOfInterest.txt

```diff
@@ -1,3 +1,4 @@
 Blue_Clams = Unionidae_Rectidentinae_Contradens_contradens, Unionidae_Rectidentinae_Solenaia_khwaenoiensis
-Green_Clams = Etheriidae_Etheriinae_Etheria_elliptica, Notina_TheLoci
-Banana_Clams = Itsa_Notthere, Notina_Theloci
\ No newline at end of file
+Green_Clams = Unionidae_Rectidentinae_Contradens_contradens, Unionidae_Rectidentinae_Solenaia_khwaenoiensis, Notina_TheLoci
+Banana_Clams = Itsa_Notthere, Notina_Theloci
+NA_Clams = Unionidae_Rectidentinae_Contradens_contradens, Unionidae_Rectidentinae_Trapezoideus_foliaceus, Unionidae_Rectidentinae_Salweeniana_salweeniana
```

Comment:
NA_Clams: contains species found in the loci taxables, but nothing in the matrix that matches colade